skflow/estimators/base.py

```diff
@@ -370,6 +372,7 @@ def save(self, path):
             if not callable(value) and value is not None:
                 params[key] = value
         params['class_name'] = type(self).__name__
+        del params['monitor']
```

Comment:
yeah, that's part of the problem - but then when you restore model, there is no way to specify monitor anymore.